reposerver/repository/repository_test.go

```diff
@@ -4325,3 +4327,45 @@ images:
 		}, res.Commands)
 	})
 }
+
+func Test_SkipSchemaValidation(t *testing.T) {
+	t.Run("helm", func(t *testing.T) {
+		service := newService(t, "testdata/broken-schema-verification")
+
+		// Fill the manifest request with as many parameters affecting Helm commands as possible.
+		q := apiclient.ManifestRequest{
+			AppName: "test-app",
+			Repo:    &argoappv1.Repository{},
+			ApplicationSource: &argoappv1.ApplicationSource{
+				Path: ".",
+				Helm: &argoappv1.ApplicationSourceHelm{
+					SkipSchemaValidation: true,
+				},
+			},
+		}
+
+		res, err := service.GenerateManifest(context.Background(), &q)
+
+		require.NoError(t, err)
+		assert.Equal(t, []string{"helm template . --name-template test-app --include-crds --skip-schema-validation"}, res.Commands)
+	})
+	t.Run("helm", func(t *testing.T) {
+		service := newService(t, "testdata/broken-schema-verification")
+
+		// Fill the manifest request with as many parameters affecting Helm commands as possible.
```

Comment:
```suggestion
```

I think this comment isn't relevant here.

Reply:
Removed